### ISSUE TITLE
Autocomplete box length (request)

### ISSUE THREAD

**Issue by ChevronColon3** [sharelatex/sharelatex#452]
The autocomplete function for ` \cite{} ` is wonderful, and saves me so much time. Thanks for implementing it. 

If the font size (Menu => Font Size) is high enough that "Press CTRL-Space to Search" (when typing `\cite{`) is forced on to two lines, it makes the suggested auto complete options harder to read (e.g: Font 20px on 1920x1080px, Font 24px on 1440x900px).

Also, if you have multiple reference keys which are very similar for the first few characters, the limited width of the suggestion box makes them difficult to differentiate

In short: If the autocomplete box could be made wider that would be pretty cool.

**Comment by zeigerpuppy** [comment]
this feature doesn't seem to work in my docker install of sharelatex, is it active in the community version?

**Comment by cmcisaac6** [comment]
The behaviour is different now, therefore closing. If you still have any issue, please re-open with an explanation or create a new issue.
